app/controllers/location_shifts_controller.rb

```diff
@@ -46,13 +46,13 @@ def destroy
 
     @location_shift.destroy
 
-    redirect_to location_shift_path(location_id: @location_shift.location_uid)
+    redirect_to location_shift_path(organisation_id: @location_shift.organisation_id)
   end
 
   private
 
-  def location
-    find_organisation_by_uid(uid: params[:location_id])
+  def organisation
+    Organisation.find(params[:organisation_id])
   end
 
   def location_shift
```

Comment:
The complication here is that we have a `Shift` model and a `LocationShift` presenter style model too, and different ones are used in the `create` and `update` actions. I think the `LocationShift` model, was a helpful wrapper around the `location_uid` thing and seeing as that's gone it might be worth ditching. I'll look into it now.